src/main/kotlin/com/github/mitchellolsthoorn/testgenie/toolwindow/TestGenieToolWindow.kt

```diff
@@ -0,0 +1,76 @@
+package com.github.mitchellolsthoorn.testgenie.toolwindow
+
+import com.intellij.openapi.ui.Messages
+import com.intellij.openapi.wm.ToolWindow
+import java.awt.Dimension
+import java.awt.event.ActionEvent
+import javax.swing.JButton
+import javax.swing.JComboBox
+import javax.swing.JComponent
+import javax.swing.JLabel
+import javax.swing.JPanel
+import javax.swing.JTextField
+
+/**
+ * This class stores the UI of the TestGenie tool window.
+ */
+class TestGenieToolWindow(_toolWindow: ToolWindow) {
+
+    private val toolWindow = _toolWindow
+
+    private var maxSizeTextField: JTextField? = null
+    private var globalTimeOutTextField: JTextField? = null
```

Comment:
Seems like those variables are repeated from the settings variables. This makes sense. I wonder, do you think it would be possible to link this tool window to the settings? So that changes here are also represented in the settings. I would think that this has something to do with persistence and what files we use to load data. 
Or are they already linked? 

P.S. 
I think I understood what GlobalTimeout is, it's just a parameter. 😅

Reply:
You do have a point. I also thought about linking them somehow. It will indeed be a hassle with synchronising their values. However, I haven't looked into it yet.